pypdf/_xobj_image_helpers.py

```diff
@@ -29,9 +29,9 @@
 
 
 try:
-    from PIL import Image
-except ImportError:
-    raise ImportError(
+    from PIL import Image, UnidentifiedImageError  # noqa: F401
+except ImportError:  # pragma: no cover
```

Comment:
This should be covered, but is just some issue with coverage measurement. We have the subprocess test which runs with `Pillow` disabled, thus we do not need the pragma here.

Corresponding PR which only seems to work locally at the moment: #2556.

Reply:
I propose to leave #noqa for the moment. and try to find a solution in #2556

Reply:
mark it resolved if ok for you

Reply:
I disagree with this - we had it without the pragma for the whole time, thus it should not have a real side effect.

Reply:
OK I've reverted,